src/Workspaces/Core/Portable/Classification/EmbeddedLanguages/PredefinedEmbeddedLanguageClassifierNames.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Classification
+{
+    internal static class PredefinedEmbeddedLanguageClassifierNames
+    {
+        /// <summary>
+        /// A special built-in classifier for classifying escapes in strings and character literals if no other embedded
```

Comment:
Would be nice if there was an `[Order(After = OrderAttribute.AllOthers)]` or something, and then this could be "just" another classifier, and be given a better name (`EscapeCharacterClassifier` perhaps)

Reply:
I'm actually even removing this fromthis list, and we'll just have this baked in as something special :)